packages/atomic/src/components/search/search-box-suggestions/atomic-search-box-instant-results/atomic-search-box-instant-results.tsx

```diff
@@ -24,6 +24,12 @@ import {
   ResultDisplayLayout,
 } from '../../../common/layout/display-options';
 import {Button} from '../../../common/button';
+import {Bindings} from '../../atomic-search-interface/atomic-search-interface';
+
+export type AriaLabelGenerator = (
+  bindings: Bindings,
```

Comment:
I'm using the **search** bindings specifically, and not just `i18n`, since I don't want to add extra parameters if developers ever need more, and bindings contain the engine, which is useful.

Reply:
Also, the instant results component uses the search bindings currently, it's not compatible with other use cases than search.